tests/signals_post_constraints/statistics

```diff
@@ -0,0 +1,27 @@
+# 1: number of constraints
```

Comment:
Yes, nothing we can do about this. It just so happens that the slot is the first place that puts anything into the table, and so it gets column 1. The TableHandler does not allow control over where a column is placed.